core/domain/interaction_jobs_one_off.py

```diff
@@ -193,13 +193,14 @@ def map(item):
                     fail_on_validation_errors=True
                 )
             except Exception as e:
-                error_messages.append(
-                    '%s: %s' % (state.interaction.id, python_utils.UNICODE(e)))
+                error_messages.append(u'%s: %s' % (state.interaction.id, e))
```

Comment:
Hm, I don't think "u" is the way we usually do it. Normally we do something like .encode('utf-8'). Could you please do that and test manually?

Reply:
Done.